src/Merchants/Resources/views/page/merchant/detail.html.twig

```diff
@@ -30,6 +32,18 @@
                     {% endif %}
                 </div>
             </div>
+
+            <h4>{{ 'merchant.addressInformation'|trans }}</h4>
```

Comment:
This is optional. We should make here an if condition to now show the headline

Reply:
Is this public? It's not part of the merchant-portal profile page